tfe.go

```diff
@@ -863,7 +863,7 @@ func decodeErrorPayload(r *http.Response) ([]string, error) {
 	errPayload := &jsonapi.ErrorsPayload{}
 	err := json.NewDecoder(r.Body).Decode(errPayload)
 	if err != nil || len(errPayload.Errors) == 0 {
-		return errs, fmt.Errorf(r.Status)
+		return errs, fmt.Errorf("%s", r.Status)
```

Comment:
For consistency, it seems like this line should be `return errs, errors.New(r.Status)` since no formatting is done and r.Status is already a string (I think)